book/accessibility.md

```diff
@@ -44,7 +44,7 @@ software.
 
 The reasons for customizing, of course, are as diverse as the customizations
 themselves. MDN [reports][mdn-def] that the World Health Organization
-[found][who-fact-sheet] as much as 15% of the world population have some
+[found][who-fact-sheet] that as much as 15% of the world population have some
```

Comment:
The edit is fine but why do we say what MDN reports about what the WHO found? I checked the primary source (the "WORLD REPORT ON DISABILITY", Dec 2011, it's the first one so frequency unclear, https://www.who.int/publications/i/item/9789241564182) and it does (after a lot of qualifying) give a number of 15.6% (page 51). I say we avoid the double reference.